addons/bus/static/tests/helpers/mock_server/models/ir_websocket.js

```diff
@@ -11,7 +11,10 @@ patch(MockServer.prototype, 'bus/models/ir_websocket', {
      * @param imStatusIdsByModel
      */
      _mockIrWebsocket__updatePresence(inactivityPeriod, imStatusIdsByModel) {
-        this._mockBusBus__sendone(this.currentPartnerId, 'bus/im_status', this._mockIrWebsocket__getImStatus(imStatusIdsByModel));
+        const imStatusNotifications = this._mockIrWebsocket__getImStatus(imStatusIdsByModel);
+        if (Object.keys(imStatusNotifications).length) {
```

Comment:
`.length > 0` (easier to see a proper boolean expression rather than rely on truthy/falsy)